pylint/__init__.py

```diff
@@ -11,22 +11,22 @@
 
 import os
 import sys
-from typing import List, Optional
+from typing import Optional, Sequence
 
 from pylint.__pkginfo__ import __version__
 
 # pylint: disable=import-outside-toplevel
 
 
-def run_pylint(*, arguments: Optional[List[str]] = None):
+def run_pylint(argv: Optional[Sequence[str]] = None):
     """Run pylint
 
-    Arguments can be a list of strings normally supplied as arguments on the command line
+    argv can be a list of strings normally supplied as arguments on the command line
```

Comment:
```suggestion
    argv can be a sequence of strings normally supplied as arguments on the command line
```
Hmm 🤔 It doesn't need to be a list, although it obviously can. E.g. a tuple would also be fine.